### DISCUSSION TITLE
i got an error when i try to create a filter for my customers in other tab

### DISCUSSION THREAD

**Discussion by IPolonio** [serenity-is/Serenity#7114] [Help]
### Before submitting the bug report, please read and check the following items

- [X] This isn't a feature request or a discussion topic (Please use discussions for that)
- [X] I have searched open and closed issues for duplicates
- [X] I have searched the documentation on the Serenity Platform website https://serenity.is/docs/
- [X] I have read the frequently asked questions https://serenity.is/docs/startsharp/faq
- [X] I have searched my problem in the discussions and the old issues https://github.com/serenity-is/Serenity/discussions

### What happened?

![image](https://github.com/serenity-is/Serenity/assets/122845674/6e5069c4-42e4-45e8-bf56-ae4d2fc0299e)

### What did you expect to happen?

that when i select a customer al the orders are placed for that customer 

### How to reproduce?

namespace MovieTutorial.CustomerPortal {
    @Serenity.Decorators.registerEditor()
    export class OrderDetailEdit extends Serenity.EntityGrid<OrderDetailsRow, any> {
        protected getColumnsKey() { return "CustomerPortal.OrderDetails"; }
        protected getDialogType() { return OrderDetailsDialog; }
        protected getIdProperty() { return OrderDetailsRow.idProperty; }
        protected getLocalTextPrefix() { return OrderDetailsRow.localTextPrefix; }
        protected getService() { return OrderDetailsService.baseUrl; }

        public _personID: number;
        private _customerID: number;

        get customerID() {
            return this._customerID;
        }

        set customerID(value: number) {
            if (this._customerID != value) {
                this._customerID = value;
                this.setEquality(OrderDetailsRow.Fields.OrderCustomerId, value);
                this.refresh();
            }
        }

        

        get personID() {
            return this._personID;
          
        }

        set personID(value: number) {
            if (this._personID != value) {
                this._personID = value;
                this.setEquality(OrderDetailsRow.Fields.OrderId, value);
                console.log(OrderDetailsRow.Fields.OrderId, value);
                console.log(this.personID);
                this.refresh();

            }
            console.log(this.personID);
        }
        
        constructor(container: JQuery) {
            super(container);
        }

        protected getButtons() {
            return null;
        }

        protected getInitialTitle() {
            return null;
        }

        protected usePager() {
            return false;
        }

        protected getGridCanLoad() {
            return this.personID != null;
            
        }
    }
}
namespace MovieTutorial.CustomerPortal {

    @Serenity.Decorators.registerClass()
    export class CustomersDialog extends Serenity.EntityDialog<CustomersRow, any> {
        protected getFormKey() { return CustomersForm.formKey; }
        protected getIdProperty() { return CustomersRow.idProperty; }
        protected getLocalTextPrefix() { return CustomersRow.localTextPrefix; }
        protected getNameProperty() { return CustomersRow.nameProperty; }
        protected getService() { return CustomersService.baseUrl; }
        protected getDeletePermission() { return CustomersRow.deletePermission; }
        protected getInsertPermission() { return CustomersRow.insertPermission; }
        protected getUpdatePermission() { return CustomersRow.updatePermission; }

        protected oeds: OrderDetailEdit
        protected afterLoadEntity() {
            super.afterLoadEntity();
            this.oeds.customerID = this.entityId;
        }
    }
}

### What Serenity Nuget Versions are you seeing the problem on? (separated by comma)

Serenity Obsolete 2.0

**Comment by volkanceylan** [comment]
Use discussions
